PostfixCalculator/main.c

```diff
@@ -0,0 +1,132 @@
+#include "../Stack/stack.h"
+#include <stdio.h>
+#include <stdbool.h>
+#include <ctype.h>
+#include <stdlib.h>
+
+bool isEmpty(StackObject* top) {
+    if (top == NULL) {
+        return true;
+    }
+    return false;
```

Comment:
Так не пишут. Пишут так:
```suggestion
    return top == NULL;
```